src/main/java/org/shanerx/tradeshop/AdminEventListener.java

```diff
@@ -0,0 +1,168 @@
+/*
+ *     Copyright (c) 2016-2017 SparklingComet @ http://shanerx.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * NOTICE: All modifications made by others to the source code belong
+ * to the respective contributor. No contributor should be held liable for
+ * any damages of any kind, whether be material or moral, which were
+ * caused by their contribution(s) to the project. See the full License for more information
+ */
+
+package org.shanerx.tradeshop.admin;
+
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.Sign;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.ItemStack;
+import org.shanerx.tradeshop.TradeShop;
+import org.shanerx.tradeshop.Utils;
+
+public class AdminEventListener extends Utils implements Listener {
+	
+	private TradeShop plugin;
+	
+	public AdminEventListener(TradeShop instance) {
+		
+		plugin = instance;
+		
+	}
+	
+	@SuppressWarnings("unused")
+	@EventHandler
+	public void onBlockBreak(BlockBreakEvent event) {
+		Player player = event.getPlayer();
+		Block block = event.getBlock();
+		
+		if (event.getBlock().getType() == Material.SIGN_POST || event.getBlock().getType() == Material.WALL_SIGN) {
+			Sign s = (Sign) event.getBlock().getState();
+			
+			if (!("[iTrade]".equalsIgnoreCase(ChatColor.stripColor(s.getLine(0))) && "[Trade]".equalsIgnoreCase(ChatColor.stripColor(s.getLine(0))) && player.hasPermission(getAdminPerm()))) {
```

Comment:
Am I wrong, or should `` &&`` be replaced by ``||`` - if it's a valid trad sign, it will evaluate to true, thus skip it. It will never be both trade and itrade.